autodispose/src/test/java/com/uber/autodispose/AutoDisposeSubscriberTest.java

```diff
@@ -179,6 +185,25 @@
     assertThat(errors.get(0)).isInstanceOf(LifecycleNotStartedException.class);
   }
 
+  @Test public void autoDispose_withProviderAndPlugin_withoutStarting_shouldFailPlugin() {
+    final RecordingObserver<OutsideLifecycleException> errorHandler = new RecordingObserver<>();
+    AutoDisposePlugins.setOutsideLifecycleHandler(new Consumer<OutsideLifecycleException>() {
+      @Override
+      public void accept(OutsideLifecycleException e) throws Exception {
+        errorHandler.onNext(e);
+      }
+    });
+    BehaviorSubject<Integer> lifecycle = BehaviorSubject.create();
+    TestSubscriber<Integer> o = new TestSubscriber<>();
+    LifecycleScopeProvider<Integer> provider = TestUtil.makeLifecycleProvider(lifecycle);
+    Flowable.just(1)
+            .to(new FlowableScoper<Integer>(provider))
+            .subscribe(o);
+
+    o.assertNotSubscribed();
```

Comment:
@hzsweers any idea why this would fail? This holds true for all the other types but for Flowable, it doesn't seem like it's the case.